.taskcluster.yml

```diff
@@ -24,9 +24,15 @@ tasks:
           go get -t github.com/taskcluster/taskcluster-cli/... 
           && cd  /go/src/github.com/taskcluster/taskcluster-cli 
           && git fetch {{ event.head.repo.url }} {{ event.head.ref }} 
-          && git checkout {{ event.head.sha }} 
           && make 
-          && go test ./...
+          && go get -u github.com/alecthomas/gometalinter 
+          && gometalinter --install --force
+          && go install ./...
+          && git checkout {{ event.head.sha }} 
```

Comment:
You should checkout before running `go install` or `make`, in case the dependencies changed.